content/tutorials/channel-rewind.textile

```diff
@@ -14,6 +16,8 @@ languages:
   - javascript
 level: easy
 reading_time: 5
+tags:
+    - Channel rewind
```

Comment:
Can we please replace all occurrences of the tag name 'Channel rewind' with 'Channel Rewind'

Reply:
Done (x1)